test/suite/hook_disagg.fail

```diff
@@ -31,7 +30,6 @@ test_cursor21.py
 test_debug_mode08.py
 test_drop.py
```

Comment:
we can skip `test_bug019` too?

Reply:
`test_hs30` and `test_stat_log01` still fail after this change, so their failures are different

Reply:
same with `test_log03` here

Reply:
cool as long as you have checked. :-)